### ISSUE TITLE
hiding external files does not work

### ISSUE THREAD

**Issue by sir-marc** [nextbigsoundinc/stylemark#39]
The possibility to hide inline 'files' works just fine, but when trying to hide an external file it doesn't really work properly.
For example this works just fine:
```
```types.css hidden
body {
  background-color: red;
}```
```
whereas this does not work:
```
```types:../lib/style.css hidden```
```
It justs displays the content of the file in plain text inside the markup, see attached screenshot
![bildschirmfoto 2018-06-07 um 17 41 43](https://user-images.githubusercontent.com/12906885/41110602-1a8a1b58-6a7a-11e8-8e2d-f15d6233e1d5.png)

**Comment by mpetrovich** [comment]
Thanks for reporting! We'll fix this soon.

**Comment by mpetrovich** [comment]
@sir-marc What version of Stylemark are you using? This works in v3.0 and was likely fixed in [v1.0.1](https://github.com/nextbigsoundinc/stylemark/releases/tag/v1.0.1). Please upgrade to v2.x or greater and retest (v1.x is deprecated).

**Comment by StuckiSimon** [comment]
@mpetrovich I've setup a quick repo using **3.0.0** to reproduce the issue: [Stylemark Issue #39](https://github.com/StuckiSimon/stylemark-issue-39).

The generated stylemark documentation is available under: [Demo](https://stuckisimon.github.io/stylemark-issue-39/)

**Comment by mpetrovich** [comment]
This is was a weird one. When the external stylesheet has an empty newline at the end of the file, it works. But when it doesn't (in your case), it's not being handled correctly. Fixing now.

**Comment by mpetrovich** [comment]
This is fixed in [v3.0.1](https://github.com/nextbigsoundinc/stylemark/releases/tag/v3.0.1)

**Comment by mpetrovich** [comment]
FYI in case it's relevant, along the way I discovered a bug with the way external wildcard references are handled: https://github.com/nextbigsoundinc/stylemark/issues/41
